src/Microsoft.Identity.Web/TokenAcquisition.cs

```diff
@@ -1093,6 +1093,9 @@ private static bool AcceptedTokenVersionMismatch(MsalUiRequiredException msalSer
         {
             switch (level)
             {
+                case Client.LogLevel.Always:
```

Comment:
Since the logging extensions have no concept of "Always", I set it to log at the lowest logging level which is critical.